app/src/main/java/bankingapp/App.java

```diff
@@ -1,5 +1,13 @@
-/*
- * Copyright Fredrick Kamau
+/**
+ * A simple banking application that allows users to create a new account and log into an existing account.
+ * The application uses SQLite as its database and includes helper classes for creating and manipulating the database,
+ * as well as classes for creating and logging into user accounts.
+ *
+ * This program takes one command line argument, which should be the URL of the SQLite database file to use.
+ *
+ * @author Fredrick Kamau
+ * @version 2.0
+ * @since 2022
```

Comment:
Neat-o!

BTW, `@since` is meant to indicate the earliest version of your software which includes and/or supports the `App` class. Obviously, every version had a `main`, so it's not as useful here.

Where it is useful is, for example, if you add new members to the `IDBTransaction` interface, and your developers want to know, what's the earliest version of IDBTransaction that supports a given method or feature?

For example, earlier today I needed to answer the question, _What's the earliest verison of Java that will compile this app?_ My crusty old version of Java did not support `nextLong(bound)` as you use it in `CreateAccount.setAccountNumber`. Once I learned which interface it lived in, it was a simple matter to find [the right answer](https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/util/random/RandomGenerator.java#L105). The linked line let me know that I should not expect to see any part of that interface before Java 1.17.